console/src/assets/i18n/en.json

```diff
@@ -52,7 +52,7 @@
             "DETAIL": "Detail",
             "CREATE": "Create",
             "MY": "My Informations",
-            "LOGINNAMES":"Login names",
+            "LOGINNAMES":"Methods to access your profile",
```

Comment:
Does this make sense?
I don't understand this one 😁

Reply:
its a the card header title under userinfo to display methods to access you profile.
We should keep it login names as long as there are no other methods like an authenticator app 👍